pkg/agent/flexvolume/server.go

```diff
@@ -161,3 +179,12 @@ func copyFile(src, dest string) error {
 	}
 	return destFile.Sync()
 }
+
+func checkIfKubeletRestartRequired(context *clusterd.Context) {
+	kubeVersion, err := k8sutil.GetK8SVersion(context.Clientset)
+	if err != nil || kubeVersion.LessThan(version.MustParseSemantic(serverVersionV180)) {
+		logger.Warning("NOTE: The Kubelet must be restarted on this node since this pod appears to " +
```

Comment:
not: for long, multiple line strings, i would use backticks "`"

Reply:
edit: nit

Reply:
i use backticks for strings when i want them to be raw and not have to escape anything.  in this case, I want the log statement to be printed as a single line, but I'm doing concatenation of a few strings solely to wrap the code and prevent it from going too wide.

are backticks (raw string) important for this? i don't see much of a difference in this specific scenario personally.